server/routes/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable eol-last */
+import {
+  Router,
+} from 'express';
+import userRoutes from './user';
+import meetupRoutes from './meetups';
+import questionsRoute from './questions';
+// import commentRoutes from './routes/comments';
+
+const router = new Router();
+
+router.use('/auth', userRoutes);
+router.use('/meetups', meetupRoutes);
+router.use('/questions', questionsRoute);
+
```

Comment:
How is the comment route used when it is not mapped here?

Reply:
The comment route is not actually functional here i was just doing some code scribbling i was supposed to remove the code for the comments